### ISSUE TITLE
Looking for contributors!

### ISSUE THREAD

**Issue by kodie** [kodie/gravityforms-repeater#75]
Since I don't really work with Wordpress much anymore I've kind of lost touch with this project. I've put in a ton of work in the past year and obviously people find it very useful (minus all the bugs) so I would really hate to see this project die.

I would absolutely love to see more people contributing to the Gravity Forms Repeater Add-On. So let me know if you'd like to help and I'll get you added as a collaborator to the official repo!

Thanks!

**Comment by lukecav** [comment]
Do you still need contributors?

**Comment by kodie** [comment]
@lukecav Yes! If you're interested, I'll get you added! I know that you've been involved with the plugin for a little while already.

**Comment by terence1990** [comment]
Hey Kodie,

Can you add me as a contributor, there are loads of bugs from recent updates as you can see from the latest Issues on this repo. I've been fixing a few on my development code, for now I can send pull requests.

Many thanks,

**Comment by kodie** [comment]
Hey @terence1990, if you could get a couple of pull requests in first, just to show that you'll be a good contributor for the community's sake, then I would be happy to make you a contributor.

**Comment by sniffles7** [comment]
Hi everyone, is this project still running? The latest updates were made months ago. I would really like to use it for my forms, but it's still buggy, especially when using file uploads. Multiple upload doesn't work at all. Thanks!
